src/common/pubsub.ts

```diff
@@ -133,10 +134,35 @@ export const notifyUserReputationUpdated = async (
     reputation,
   });
 
-export const notifyAlertsUpdated = async (
+export const removeSomeProps = <T>(obj: T, excludes: (keyof T)[]): Partial<T> =>
+  Object.keys(obj).reduce((result, key) => {
+    if (excludes.some((excluding) => excluding === key)) {
+      return result;
+    }
+
+    return { ...result, [key]: obj[key] };
+  }, {});
+
+export const notifyAlertsUpdated = (
   log: EventLogger,
   alerts: ChangeObject<Alerts>,
-): Promise<void> => publishEvent(log, alertsFilterUpdatedTopic, alerts);
+): Promise<void> =>
+  publishEvent(log, alertsUpdatedTopic, removeSomeProps(alerts, ['userId']));
```

Comment:
Why do you remove `userId`? It's bad practice removing props of existing topics. It might break the subscribers.

Reply:
I was thinking that we always embed the `userId` in our message, but I agree it is not a good idea to reshape existing messages.

Reply:
And also `userId` is not embedded anywhere else if you don't provide it in the payload